src/travel-details-screens/components/Trip.tsx

```diff
@@ -157,7 +157,7 @@ const useStyle = StyleSheet.createThemeHook((theme) => ({
     paddingTop: theme.spacings.medium,
   },
   container: {
-    paddingBottom: theme.spacings.medium,
+    paddingVertical: theme.spacings.medium,
```

Comment:
Before this change:

<img width="495" alt="image" src="https://github.com/AtB-AS/mittatb-app/assets/43450882/bc5c1f5a-fdac-4127-a481-5db7425f122e">

After the change:

<img width="495" alt="image" src="https://github.com/AtB-AS/mittatb-app/assets/43450882/30a2af24-ab89-4d0a-9150-4aa6ba20dd3a">

Reply:
It was fixed in another branch